### ISSUE TITLE
Allow node titles to be available in layout contexts

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#4497]
This came up as a limitation in #3750

@olafgrabienski (re breadcrumb blocks):

>The block itself works but "the current page title as the last breadcrumb" (enabled in block settings) is missing. I get an empty span instead:
```
<ol>
  <li><a href="/">Home</a></li>
  <li><span></span></li>
</ol>
```

@docwilmot 

>I think I understand your point. You're expecting that the page title of a context node will be available as a title block or a breadcrumb option. That isn't the case, since page title isn't a field.

@olafgrabienski 

>I see! What would be the use case of custom contexts when important data like node titles or let's say user names and user pictures aren't available? Or do you see a chance to tie non-field data to custom contexts in a follow-up?

@docwilmot 

>Contexts simply make the data available to blocks on the layout. It's up to developers to code blocks that use the info. Field blocks are already coded to use context data. Title blocks, not yet.

**Comment by klonos** [comment]
PS: this seems to be related to #933 (or would be moot if that was implemented, as the node title would become a normal field then).

**Comment by olafgrabienski** [comment]
> would be moot if that was implemented, as the node title would become a normal field then

Yes, **if** it was implemented... Looks like #933 was postponed to 2.x.
